packages/components/src/components/hds/form/text-input/field.ts

```diff
@@ -39,12 +39,12 @@ export interface HdsFormTextInputFieldSignature {
 }
 
 export default class HdsFormTextInputField extends Component<HdsFormTextInputFieldSignature> {
-  @tracked isPasswordMasked = true;
-  @tracked type;
+  @tracked private _isPasswordMasked = true;
+  @tracked private _type;
```

Comment:
I believe this is a good example to reflect on what we mean by private.

Generally, all class properties can be considered private, as consumers are not meant to use them directly. While changing all of them to private is accurate, it will not benefit us or our consumers.

Where I believe it's important to make a distinction is on properties that do not mirror arguments 
 - `type` stores [a public argument](https://helios.hashicorp.design/components/form/text-input?tab=code#formtextinputfield-1)
 - `_isPasswordMasked` is used for an internal state

```suggestion
  @tracked private _isPasswordMasked = true;
  @tracked type;
```